### PR TITLE
Fix asset_lint job

### PR DESCRIPTION
**Pull request by szepeviktor**
@dingo-d

### CODE REVIEW

File: .github/workflows/quality.yml [hunk 1]
```diff
@@ -186,6 +186,11 @@ jobs:
         uses: "actions/setup-node@v3"
         with:
           node-version: "${{ matrix.node }}"
-      - run: |
-          npm install
-          gsnpm run lint
```
Comment (szepeviktor) [inline .github/workflows/quality.yml:191]: There is no "gsnpm" command.

File: .github/workflows/quality.yml [hunk 2]
```diff
@@ -186,6 +186,11 @@ jobs:
...
+
+      - name: "Install dependencies"
+        run: |
+          npm ci
```
Comment (szepeviktor) [inline .github/workflows/quality.yml:192]: "ci" (clean install) does not modify the lock file.

File: .github/workflows/quality.yml [hunk 3]
```diff
@@ -186,6 +186,11 @@ jobs:
...
+
+      - name: "Run checks"
+        run: |
+          npm run lint
```
Comment (szepeviktor) [inline .github/workflows/quality.yml:196]: Let's have each step separately.

**Comment by szepeviktor** [comment]
image

**Comment by dingo-d** [comment]
Thanks for the fixes. I've changed the branch to `develop`. Can you just rebase your patch so that the conflicts are gone?

**Comment by szepeviktor** [comment]
Done 🍏

**Review by dingo-d — APPROVED** [review]

**Pull request merged**